packages/web3-utils/README.md

```diff
@@ -39,12 +41,20 @@ console.log(Utils);
 }
 ```
 
+Import what you need
+
+```js
+import { asciiToHex } from 'web3-utils';
+
+console.log(asciiToHex('I have 100!'));
+> "0x49206861766520313030e282ac"
+```
+
 ## Types 
 
-If you are using TypeScript all the types are defined in the `index.d.ts` file
+If you are using TypeScript then all types are defined within the package itself so no need
+to install any `@types` libraries. Just import the library and the types should be exposed.  
```

Comment:
I think it's not necessary to say that they shouldn't install the @types lib of web3. Just leave the sentence as it was before.

Reply:
I think if we do not say this explicitly we will get a lot of support cases asking about this, this is why i added it in. If you need want me to remove i will 👍

Reply:
Yep remove it. If this will be the case we can add this to the documentation.